pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -305,6 +311,9 @@ func ValidateNodeResourcesFitArgs(path *field.Path, args *config.NodeResourcesFi
 	}
 
 	if args.ScoringStrategy != nil {
+		if !supportedScoringStrategyTypes.Has(string(args.ScoringStrategy.Type)) {
```

Comment:
I think we need to add context here to the path to include ScoringStrategy as a parent to the validations below.

```
strategyPath := path.Child("scoringStrategy")
```

and then use `strategyPath` instead of `path` below.

Reply:
Thanks @ahg-g  that is reasonable. I have updated the PR.